<launch>	
	<node pkg="performance_tests" type="py_talker.py" name="py_talker">
		<param name="hertz" type="int" value="1800"/>
	</node>
	
	<node pkg="performance_tests" type="cpp_listener" name="cpp_listener"/>
</launch>
